### ISSUE TITLE
Option for existing ingress tls secret

### ISSUE THREAD

**Issue by pandalec** [Mailu/helm-charts#192]
**Is your feature request related to a problem? Please describe.**
I am currently using an existing wildcard certificate inside my cluster and currently it is not possible to use own certificates for ingress

**Describe the solution you'd like**
Add an optional variable like `ingress.secretName` for ingress.yaml, e.g.
```
{{- if .Values.ingress.secretName }}
  tls:
  - secretName: {{ .Values.ingress.secretName }}
    hosts:
{{- range .Values.hostnames }}
    - "{{ . }}"
{{- end }}
{{- else }}
  tls:
    {{- if .Values.admin.persistence.storageClass }}
    storageClassName: {{ .Values.admin.persistence.storageClass }}
    {{- end }}
  - secretName: {{ $fullname }}-certificates
    hosts:
{{- range .Values.hostnames }}
    - "{{ . }}"
{{- end }}
{{- end }}
```

Best regards,
parsi

**Comment by fastlorenzo** [comment]
This should be addressed in #193

**Comment by fastlorenzo** [comment]
Fixed in 1.0.0
